h/groups/views.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+
+import deform
+import colander
+import hashids
+from pyramid import httpexceptions as exc
+from pyramid.view import view_config
+
+from h.groups import schemas
+from h.groups import models
+from h.accounts import models as accounts_models
+from h import security
+
+
+def _get_hashids(request):
+    salt = security.derive_key(
+        request.registry.settings["secret_key"], "h.groups.hashid", length=20)
```

Comment:
As we discussed, I think this is a dangerous route to go down. Just add a config option for this.